src/components/fetch-paginator.js

```diff
@@ -141,26 +163,27 @@ input:checked + poke-card {
             const isChecked = this.#checkedsPokemons.has(pokeName);
             return html`
 <label>
-<input type="checkbox"
+<input @change=${this.#handleChange}
+       type="checkbox"
        value=${pokeName}
-       .checked=${isChecked}>
+       ?checked=${isChecked}>
   <poke-card .pokemon=${poke}></poke-card>
 </label>
-`;
+`
         };
 
         return this._pokemons.map(render_by_pokemon);
     }
 
     render() {
         return html`
-<section @change=${this.#handleChange}  class="poke-container">
-         ${this.#mapPokemonsToHTML()}
+<section class="poke-container">
+   ${this.#mapPokemonsToHTML()}
 </section>
-<section  class="btns-container" @click=${this.#handleClick}>
-<button value="prev" ?disabled=${this.#prevUrl === ""}>Go prev</button>
-<button value="home">Go home</button>
-<button value="next" ?disabled=${this.#nextUrl === ""}>Go next</button>
```

Comment:
The same as [above](https://github.com/Bihart/for-fun-lit-elements/pull/1/files#r1032800163)